app/assets/stylesheets/sulCollection.scss

```diff
@@ -137,6 +137,10 @@
   }
 }
 
+.al-request-button {
+  border-radius: 0;
```

Comment:
Should this set `--bs-btn-border-radius: 0` instead?

Reply:
This seems like it might be shared with https://github.com/sul-dlss/stanford-arclight/blob/1dc44ffad79b7c670f462d935d781a10f5afbd8f/app/assets/stylesheets/sulCollection.scss#L65-L67

Reply:
So maybe we kick up `.btn-sm` to the top level (un-nest it)?

Reply:
Good catch. That makes sense to me.